operator/src/main/java/org/keycloak/operator/Constants.java

```diff
@@ -71,4 +72,8 @@ public final class Constants {
     public static final String CACHE_CONFIG_FOLDER = CONFIG_FOLDER + "/" + CACHE_CONFIG_SUBFOLDER;
 
     public static final String KEYCLOAK_HTTP_RELATIVE_PATH_KEY = "http-relative-path";
+
+    // Container requests/limits resources (tested suitable minimum for well-performant Keycloak)
+    public static final Quantity KEYCLOAK_RESOURCES_MEMORY_REQUESTS_DEFAULT = new Quantity("768Mi");
+    public static final Quantity KEYCLOAK_RESOURCES_MEMORY_LIMITS_DEFAULT = new Quantity("4Gi");
```

Comment:
@vmuzikar I've been thinking about it, and I'm not sure if that's necessary. In the initial version of this PR, I put the capability to change these defaults, but then, I realized that it might not be necessary at all. Do we need to provide other config options for this purpose? 

To be more specific, when the user wants to set different values, the `limits/requests` should be used for this purpose. Why should different properties be set for it when the Operator sets it all for these managed containers? 

@vmuzikar Or do you see any benefit for doing it?

Reply:
Yes, the defaults are not meant to be changed by the user. But IMHO we should still have it configurable as an escape hatch (if for whatever reason the defaults need to be different for that particular Operator installation). This would also align with other default values like Keycloak image coordinates.

Reply:
@vmuzikar ok, works for me